fdbserver/DeltaTree.h

```diff
@@ -26,6 +26,9 @@
 #include "fdbserver/Knobs.h"
 #include <string.h>
 
+#define deltatree_printf(...)
```

Comment:
I'll update this, I was being hacky because this is temporary.  I want to refactor Redwood's debug stuff into one place, with independently toggled scopes for the different components/contexts - Pager, FIFOQueue, BTreeCommit, BTreeCursor, DeltaTree.

Reply:
Done.